FirebaseAuth/Sources/AuthProvider/OAuth/FIROAuthCredential_Internal.h

```diff
@@ -41,22 +41,27 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly, nullable) NSString *pendingToken;
 
-/** @fn initWithProviderId:IDToken:accessToken:secret:pendingToken
+/** @property fullName
+    @brief The full name of the user associated with this OAuthCredential.
+ */
+@property(nonatomic, readonly, nullable) NSPersonNameComponents *fullName;
+
+/** @fn initWithProviderId:IDToken:rawNonce:accessToken:secret:fullName:pendingToken
     @brief Designated initializer.
     @param providerID The provider ID associated with the credential being created.
     @param IDToken The ID Token associated with the credential being created.
     @param rawNonce The raw nonce associated with the Auth credential being created.
     @param accessToken The access token associated with the credential being created.
     @param secret The secret associated with the credential being created.
+    @param fullName The displayName associated with the credential being created.
```

Comment:
I think _displayName_ should be _full name_  in the description, like everywhere else.

Reply:
Done.